zeppelin-web/Gruntfile.js

```diff
@@ -487,6 +505,7 @@ module.exports = function (grunt) {
   ]);
 
   grunt.registerTask('build', [
+    'newer:jscs',
```

Comment:
this should be `jscs` only in case of build.
`newer:jscs` should be used for the watch (around line 118 for basic .js and 125 for test files).